### DISCUSSION TITLE
Docker images for 3.x

### DISCUSSION THREAD

**Discussion by hatdragon** [OpenAS2/OpenAs2App#280] [Q&A] [answered]
Hey, I see that you all have a docker image available for 2x and 3.0, are there plans on updates for 3.1/3.2 and beyond?

**Comment by igwtech** [comment, marked as answer]
https://github.com/OpenAS2/OpenAs2App/blob/9151132176fed3e553954bd7e49bdd67d7fce2e0/Server/pom.xml#L108

Creates a Zip-Bomb which stops the Docker scripts from creating a docker image.
I'm manually patching it and submitting an image for 3.2.0 into the docker hub repo
